### ISSUE TITLE
Setup lint watcher?

### ISSUE THREAD

**Issue by vanilla-james** [beoble/beoble-toolkit#81]
`packages/react/src/components/Avatar/index.tsx`

![image](https://user-images.githubusercontent.com/6829451/194389413-eec8b91e-297b-46dd-90f3-a6aaa9ca5379.png)

![image](https://user-images.githubusercontent.com/6829451/194389527-fbe61c2d-95f5-4d40-90a9-45f29f521c92.png)

**Comment by bamnenim-kryptonium** [comment]
Can you explain more about what is lint watcher?
is it more than eslint and prettier?

**Comment by vanilla-james** [comment]
> Can you explain more about what is lint watcher? is it more than eslint and prettier?

Yes, it's basically eslint with realtime updates in terminal console (like Create-React-App provides).
~ realtime watching of eslint errors and warning.

**Comment by bamnenim-kryptonium** [comment]
I see, but I think there is already a real-time watcher in my IDE as below.
I haven't set up it by myself, so i think it is basic feature provided by nx.

<img width="689" alt="스크린샷 2022-10-07 21 09 05" src="https://user-images.githubusercontent.com/95412790/194561353-aac76b19-3a89-48b4-b5a7-f445e437c100.png">
